src/amo/components/AddonReviewCard/index.js

```diff
@@ -493,6 +494,13 @@ export class AddonReviewCardBase extends React.Component<InternalProps> {
       >
         <div className="AddonReviewCard-container">
           {errorHandler.renderErrorIfPresent()}
+          {review && review.isDeleted && (
+            <Notice type="error" className="Addon-non-public-notice">
+              {i18n.gettext(
+                'This is not a public review. You are only seeing it because of elevated permissions.',
```

Comment:
Let's change the text to: 'This rating or review has been deleted. You are only seeing it because of elevated permissions.' This will help communicate exactly what happened.